src/write.c

```diff
@@ -86,6 +86,7 @@ typedef struct avifEncoderItem
     avifCodecConfigurationBox av1C;       // Harvested in avifEncoderFinish(), if encodeOutput has samples
     uint32_t cellIndex;                   // Which row-major cell index corresponds to this item. ignored on non-av01 types
     avifBool alpha;
+    avifBool hiddenImage; // A hidden image item has (flags & 1) equal to 1 in its ItemInfoEntry.
```

Comment:
Right now, `item->hiddenImage` is true iff `item->dimgFromID` is nonzero. So we could just check `item->dimgFromID != 0` and not add the `hiddenImage` field. But I think adding the `hiddenImage` field will be a more general solution.